R/indicator_native_range_year.R

```diff
@@ -0,0 +1,117 @@
+#' Create interactive plot for counts per native region and year of introduction
+#' 
+#' Based on \code{\link{countYearProvince}} plot from grofwild
+#' @param type character, native_range level of interest should be one of 
+#' \code{c("native_continent", "native_range")} 
+#' @param xlab character string, label of the x-axis. Defaults to "year".
+#' @param ylab character string, label of the y-axis. Defaults to "number of 
+#' alien species".  
+#' @inheritParams countYearProvince
+#' @return list with:
+#' \itemize{
+#' \item{'plot': }{plotly object, for a given specie the observed number 
+#' per year and per province is plotted in a stacked bar chart}
+#' \item{'data': }{data displayed in the plot, as data.frame with:
+#' \itemize{
+#' \item{'year': }{year at which the animal was introduced}
+#' \item{'nativeRange': }{native range of the introduced animal}
+#' \item{'aantal': }{counts of animals}
+#' }
+#' }
+#' }
+#' @import plotly
+#' @importFrom reshape2 melt
+#' @importFrom INBOtheme inbo.2015.colours
```

Comment:
I removed INBOtheme from trias functions as I got some troubles with it. Moreover, you don't use `inbo.2015.colours`  explicitly in the code. To use INBOtheme colors, just load INBOtheme colors before using this function. This is how I do it in Rmd files of [indicators repo](https://github.com/trias-project/indicators) where I use trias' visualization functions.

Reply:
@damianooldoni should I write a .rmd for this function as well ?